### DISCUSSION TITLE
Email notification - Unable to find the template that exists

### DISCUSSION THREAD

**Discussion by chadcrowell** [solspace/craft-freeform#907] [Questions & Answers] [answered]
**Describe the bug or issue you're experiencing**
I have a Craft 3.9.5 install with Freeform 3.13.28. This has been up and running since 2021. There are 16 unique freeform forms on the site, and each has its own admin and customer email notifications, stored as templates. Every one of these extends the same template: _emails/mainTemplate template. For 14 of the forms, notifications send without issue. For 2 of the forms, I get these errors emails from Freeform whenever someone submits the form:

```
Forms Alert
Failed to send email notification!
Forms failed to send the Main Oem Contact Form Admin email notification to [xxxxx@xxx.com](mailto:xxxxx@xxx.com) for the Main - OEM Contact Form form on Nov 3, 2023, 10:17:26 AM. The error received was:
Unable to find the template “_emails/mainTemplate” in "__string_template__87d265e916449584b3484069e96bde014d683c3a3e4e0ae2464907aa1a32fb88" at line 2.
```

As I said, the template _emails/mainTemplate exists and all other email notifications also reference it, and those send without issue. Here are the contents of some templates:

- _emails/mainTemplate.twig:  https://gist.github.com/chadcrowell/44d3b5c46c7a71579718d465232b3906
- _emails/mainOemContactFormAdmin.twig (errors each time): https://gist.github.com/chadcrowell/bd58391beb0277e0bf0aaa24eb29baf4
- _emais/mainContactFormAdmin.twig (works every time): https://gist.github.com/chadcrowell/1f70a6a902617072812c7ccf65399115
- Screenshot of the form: https://d.pr/i/fUn64d
- Screenshot of fields: https://d.pr/i/lCEpAD

**Steps to reproduce**

1. Website user submits form
2. I receive notice of error from Freeform

**Expected behavior**
Form sends to admin

**Craft & Plugin Info (please complete the following information):**

- Craft Version: 3.9.5
- Freeform Version: 3.13.28
- Freeform Edition: Pro
- Fresh Install or Upgrade: Site running since 2021, fresh install at the time

**Comment by kjmartens** [comment, marked as answer]
Hi @chadcrowell,

Every template located inside the **Email Templates** directory is expected to be a complete template (containing everything Freeform expects). If you are using anything like layout templates, includes, etc, Freeform will choke on this and display an error.

If you wish to use this approach, it's still possible, but you need to make sure those additional templates are located _outside of_ the **Email Templates** directory path. 🙂 

https://docs.solspace.com/craft/freeform/v4/overview/email-notifications/#using-layouts

If I am missing anything obvious, I apologize, as I was unable to view your template links (GitHub redirected endlessly and then timed out - not sure why).
